frontend/src/components/dashboard/page_views/GroupedTable.vue

```diff
@@ -60,18 +55,19 @@
         data () {
             return {
                 headers: [
-                    { text: '', align: 'center', value: 'page' },
-                    { text: '', align: 'center', value: 'title' },
+                    { text: '', align: 'center', value: 'page_url' },
+                    { text: '', align: 'center', value: 'page_title' },
                     { text: '', align: 'center', value: 'bounce_rate' },
                     { text: '', align: 'center', value: 'exit_rate' },
-                    { text: '', align: 'center', value: 'page_views' },
-                    { text: '', align: 'center', value: 'avg_time' },
+                    { text: '', align: 'center', value: 'count_page_views' },
                 ],
             };
         },
         computed: {
-            tableData () {
-                return this.items;
+            getTableData () {
+                return this.items.map((item) => {
```

Comment:
it seems this computed prop is useless, you can just use prop `items` in teemplate